.woke.yml

```diff
@@ -0,0 +1,9 @@
+rules:
```

Comment:
I suppose the tool doesn't look in `pyproject.toml`?

Reply:
I don't think so, it's not a python-specific project.